### DISCUSSION TITLE
Make Termux scripts accessible through FOSS Automation

### DISCUSSION THREAD

**Discussion by chribonn** [termux/termux-app#3841] [General]
Firstly I am on now way connected to Automation.  Secondly I never developed on android (I started life on COBOL Mini Computers) and today work C#, Python and similar languages.

_Apologies if what I say sounds stupid_.

I was wondering if one could fork either termux:widget or termux:tasker such that the scripts within termux can be called from a third party tool. I've used widget (wrote a how-to using termux for LogSeq at https://www.alanbonnici.com/2024/02/how-to-clone-and-synchronise-github.html) and was wondering whether the finger tap on the termux script in termux widget could be codified thereby allowing an app invite it.

Advise and thoughts appreciated. Don't want to waste time on something that is not doable in android / possible with tasker. If the idea is doable would widget or tasker be the best fork?

Thanks

**Comment by agnostic-apollo** [comment]
The Automation app needs to add support for [locale](https://www.twofortyfouram.com/) library like other apps like Tasker, etc have to be able to call termux-tasker.

Otherwise, use `RUN_COMMAND` intent if the app supports sending intents. The Automation dev will also need to add the `RUN_COMMAND` permission in the app `AndroidManifest.xml`.

https://github.com/termux/termux-app/wiki/RUN_COMMAND-Intent
